pkg/scheduler/framework/plugins/noop/noop.go

```diff
@@ -23,27 +23,28 @@ import (
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 )
 
-// NoOpFilter is a plugin that implements the filter plugin and always returns Success.
+// NoFilter is a plugin that implements the filter plugin and always returns Success.
 // This is just to make sure that all code dependencies are properly addressed while
 // working on legitimate plugins.
-type NoOpFilter struct{}
+// Note: The struct cannot be named NoOpFilter, otherwise the golint check cannot pass
+type NoFilter struct{}
```

Comment:
just name it `Filter`

Reply:
+1 to naming it Filter since the package name should give enough context.

Reply:
> +1 to naming it Filter since the package name should give enough context.

OK